nimare/meta/kernel.py

```diff
@@ -125,14 +125,16 @@ def transform(self, dataset, masker=None, return_type="image"):
                 masker is not None
             ), "Argument 'masker' must be provided if dataset is a DataFrame."
             mask = masker.mask_img
-            coordinates = dataset.copy()
+            coordinates = dataset
             assert (
                 return_type != "dataset"
             ), "Input dataset must be a Dataset if return_type='dataset'."
 
             # Calculate IJK. Must assume that the masker is in same space,
             # but has different affine, from original IJK.
-            coordinates[["i", "j", "k"]] = mm2vox(coordinates[["x", "y", "z"]], mask.affine)
+            coordinates[["i", "j", "k"]] = mm2vox(
+                dataset[["x", "y", "z"]], mask.affine
+            )  # Possibily optimize this
```

Comment:
```suggestion
            )  # TODO: Possibly optimize this
```

Reply:
Oh, sure. I forgot to remove these so I think it's probably better to just remove them.

Reply:
That sounds good. I assumed they were just things you wanted to deal with later.